Makefile

```diff
@@ -31,6 +31,12 @@ build:
 	@go build -o build/ ${LDFLAGS} ./cmd/celestia
 .PHONY: build
 
+## build-jemalloc: Build celestia-node binary with jemalloc allocator for BadgerDB instead of Go's native one
+build-jemalloc: jemalloc
+	@echo "--> Building Celestia with jemalloc"
+	@go build -o build/ ${LDFLAGS} -tags jemalloc ./cmd/celestia
+.PHONY: build-jemalloc
+
```

Comment:
Yes, and the profiles are impressive. I would keep it for users who know what they are doing, like our infra.

We would need to add this to the docs for BN runners.

Reply:
I think it would be great to make it default, if tested and stable. But install process could be troublesome for users. I think it can become default option for binary distributions versions of node